cmd/vulcan-repository-sctrl/custom-rules/repository-with-security-control.yaml

```diff
@@ -0,0 +1,100 @@
+rules:
+  - id: repository-with-lava-scan
+    message: 'Action using lava scan binary'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - pattern-regex: 'lava scan'
```

Comment:
Should we also look for `lava run`?

Reply:
You are right. I believe is less common but possible.
Added [here](https://github.com/adevinta/vulcan-checks/pull/806/files#diff-92895a02ca895131904cdd6800f5bf564cef58757c5826fdf73c9b0e18837c62R13).